x-pack/plugins/security_solution/cypress/integration/detection_alerts/alerts_detection_callouts_readonly.spec.ts

```diff
@@ -38,6 +38,7 @@ const waitForPageTitleToBeShown = () => {
 describe('Detections > Callouts indicating read-only access to resources', () => {
   const ALERTS_CALLOUT = 'read-only-access-to-alerts';
   const RULES_CALLOUT = 'read-only-access-to-rules';
+  const NEED_ADMIN_FOR_UPDATE_CALLOUT = 'need-admin-for-update-rules';
```

Comment:
Yeah, because with this role I do not want the `NEED_ADMIN_FOR_UPDATE_CALLOUT` to end up showing per the user's role and the fact that this test will return `index_mapping_outdated: false`.

The other test that is the companion to this one is:
```ts
alerts_detection_callouts_readonly_need_admin.spec.ts
```

Where it returns `index_mapping_outdated: true` and then I test to ensure that it shows up next to each of these callouts.